coinbase_HPO_test_products.py

```diff
@@ -0,0 +1,65 @@
+
+from selenium import webdriver
+import unittest
+import platform
+import os
+import config
+import secure.creds
+from pages.coinbase.homepage import HomePage
+import tools.helpers
+
+root_path = os.path.dirname(os.path.realpath(__file__))
+if 'Darwin' in platform.system():
+    chromedriver_path = config.mac_chromedriver_path
+else: 
+    chromedriver_path = config.win_chromedriver_path
+
+
+class HomePageSetup(unittest.TestCase):
+
+    def setUp(self):
+        self.driver = webdriver.Chrome(root_path + config.win_chromedriver_path)
+        self.page = HomePage(self.driver)
+    
+    def tearDown(self):
+
+        self.page.driver.quit()
+
+
+class HomePageProducts(HomePageSetup):
+    """
+    Confirm the list of the companie's featured products are present on
+    homepage. Ensure that clicking on each product link will navigate to the
+    dialogue box 'create a new account'.
+
+    acceptance cirteria
+    --------------------
+    -All of the provided featured products are listed.
+    -When clicking on a random product link it opens the "create new account
+     dialogue box".   
+    """
+
+    def testProductsExist(self):
+        #the source of truth is the provided_featured_product_set
+        provided_featured_product_set = {'Bitcoin\nBTC', 'Ethereum\nETH', 
+                                          'Bitcoin Cash\nBCH', 'Litecoin\nLTC'}  
```

Comment:
line 44 and 45 is where you originally asked me to use selenium objects instead of text. Also on line 43 and 46 I put in these comments to help make this test read better -> You want the tests to read like a set of instructions which is challenging for this test.